### DISCUSSION TITLE
How to restart the whole docker-compose file?

### DISCUSSION THREAD

**Discussion by Jorman** [containrrr/watchtower#1074] [Q&A]
Hi, I searched information about this, but I can't find any. I'm pretty new to docker, so maybe I searched in a wrong way.
Let me explain what happen:
I've one docker with 3 service inside and happen that only one service have to be upgraded, but all the three services depends on each other. So, when watchover upgrade one of the service and restart it, all the docker-compose stop working, and I've to manually down/up the docker-compose file.

There's any workaround to restart the whole docker-file each time one of the services is upgraded?

Jorman

**Comment by simonaronsson** [comment]
Hi @Jorman!

This is solvable in watchtower using labels, see https://containrrr.dev/watchtower/linked-containers/. Although, keep an eye on #1064 as that currently breaks the functionality (possibly, not verified yet).

Best,
Simme

**Comment by Jorman** [comment]
Hi @simskij 

Thank you for your suggestion, I'll try to look at your link and try to figure out how to set all.

Jorman
